corehq/sql_db/util.py

```diff
@@ -1,28 +1,21 @@
-
 import uuid
 from collections import defaultdict
+from functools import wraps
+from random import choices
 
-from django.db import OperationalError
-
-from corehq.util.datadog.utils import load_counter_for_model
-
-try:
-    from random import choices
-except ImportError:
-    from corehq.sql_db.backports import choices
```

Comment:
`corehq.sql_db.backports.choices()` can be killed too.

Reply:
What do you mean? I removed the whole file in https://github.com/dimagi/commcare-hq/pull/25228/commits/2f47777a93937dd072e99a9e3f10372881ed250d

Reply:
Sorry, I missed that.

Today I'm just going to blame GitHub:
![image](https://user-images.githubusercontent.com/708421/64033618-1c589b00-cb4d-11e9-9698-e31b3db1a6d4.png)